.github/workflows/continuous-integration.yml

```diff
@@ -16,15 +16,12 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-latest
-          - macos-latest
+          - [self-hosted, Linux, X64]
+          - [self-hosted, macOS, ARM64]
```

Comment:
I would rather you used GH hosted runners while our macOS runner is out of service. You can use [`runner.environment == 'github-hosted'`](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#runner-context) condition to enable steps that are not needed on the self-hosted ones.

Reply:
I meant for macOS builds, not the Linux ones, of course.

Reply:
If it's temporary, I'm tempted to copy/paste the workflow in a different file - I feel like adding conditional ifs in YAML (hey, if only we could use Nickel :upside_down_face: ) is going to be more trouble than doing the stupid thing, especially if it's temporary.